packages/botonic-plugin-contentful/src/contentful/delivery-utils.ts

```diff
@@ -83,13 +83,16 @@ export class ContentfulEntryUtils {
       fields.dateRange &&
       ContentfulEntryUtils.fromDateRangeEntry(fields.dateRange)
 
+    const customFields: Record<string, unknown> = {}
+
     return new CommonFields(entry.sys.id, fields.name || '', {
       keywords: fields.keywords,
       shortText: fields.shortText,
       partitions: fields.partitions,
       searchableBy,
       dateRange,
       followUp,
+      customFields,
```

Comment:
remove, since it will be automatically set to {}.
Add here a comment saying "customFields cannot be easily diferenciated from standard fields until the content is created. see addCustomFields."